app/views/posts/index.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#posts').append('<%= escape_javascript render(@posts) %>');
+$('.pagination').replaceWith('<%= escape_javascript will_paginate(@posts) %>');
```

Comment:
Try to not follow this for further tasks. Do not use ruby variables directly in js. Looks for ways to do on callbacks of ajax requests.

Reply:
Ok. Will check that